app/models/alert.rb

```diff
@@ -67,8 +67,7 @@ def email_course_admins
   def email_target_user
     return if emails_disabled?
     return if target_user.nil?
-    AlertMailer.alert(self, target_user).deliver_now
-    update_attribute(:email_sent_at, Time.now)
+    AlertMailerWorker.schedule_email(alert: self, user: target_user)
```

Comment:
I see now ... I was assuming we wanted to send *all* alert emails asynchronously.

See adjusted PR.